adr/0006-dev-discussions.adoc

```diff
@@ -0,0 +1,122 @@
+= Setup of Development Discussions  
+
+* Status: proposed
+* Date:  2024-07-18
+
+== Context and Problem Statement
+
+Currently development discussions happens in a variety of places, including the mailing list, zulip, github issues, pull-requests, discussions, calls etc.
+
+Its awesome that we have so many ways to communicate, but it also makes it difficult to find the information you need or being able to at least follow certain areas.
+ 
+This ADR proposes to have a more structured way of having development discussions, so that it is easier to find and follow such discussions.
+
+== Decision Drivers
+
+* Enable better open governance
+* Prefer async over sync communication
+* Avoid requiring to subscribe to a "raging river" of notifications that is not filterable
+* Allow use of e-mail as not everyone can/will use web-ui
+* Have consistent place to publish and lookup Quarkus development ideas/suggestions
+
+== Scenarios (optional)
+
+Quarkus core developer, where do they best post ideas for suggestion and feedback? Zulip chat? open issue ? submit PR directly ? quarkus-dev ? quarkus discussions ? private chat ? Today all of the above happens based on habbit/or specific requests.
+
+A dev interested in doing contributions by following the ongoing conversations - which of the many channels does he follow? 
+
+== Considered options (optional)
+
+Primarily use quarkus-dev mailing list::
+    Advantages::::
+        * well-known and understood by most if not all existing Quarkus contributors (~1000+ subscribers)
+        * can post, comment and read with any e-mail client
+        * filters out most noise as people need to subscribe
+    Disadvantages::::
+        * overtime it has died out, rarely where conversations starts/continues being discussed
+        * e-mail/google groups is just not where new people show up as much now
+        * requires subscribing to specific list to post
+        * spam (fixed now by moderation)
+        * arbitrary blocking of mails (specifally @gsmet and others)
+
+Primarily have "Design Discussions" a sub-category of Quarkus Community/User Discussions::
+    Advantages::::
+        * Centralized location for discussions, anyone sees it
+        * Anyone can read
+        * Anyone with GitHub account can post/reply
+        * Moderation tools available to battle spam/noise
+    Disadvantages::::
+        * Notifications via email comes for *all* discussions, not possible to choose specific category
```

Comment:
As a note, you can turn off email notifications and use the GitHub notifications panel instead. I've done this and it has been very helpful in controlling email volume.